src/wallet/rpcwallet.cpp

```diff
@@ -75,6 +75,16 @@ bool EnsureWalletIsAvailable(bool avoidException)
     return true;
 }
 
+bool EnsureHDSeedIsAvailable(bool avoidException = false) {
```

Comment:
This look weird to have a flags to control exception should raise or not. We can move the caller after help message and remove this flag.